src/app/components/Timer.js

```diff
@@ -0,0 +1,30 @@
+export const Timer = (duration) => {
+  let timer = duration;
+
+  const timerContainer = document.createElement('div');
+  timerContainer.id = 'quiz-timer';
+  timerContainer.classList.add('timer');
+
+  setInterval(() => {
+    ({ timer } = countdown(timer, timerContainer));
+  }, 1000);
+
+  return timerContainer;
```

Comment:
1) Nie mogę tego odpalić teraz (brak dostepu do komputera!).
 To działa? Liczy cały czas w dół? Wyświetla tez zmieniony wynik? To działa?
Jeśli tak to gratki, bo sprytnie zrobione!

2) Dlaczego 'timer' jest w nawiasach { } ? Nie może być bez? (pytam bo nie wiem)

Reply:
1. działać działa, ale może być lepiej tak jak napisał Mateusz
2. bo zapomniałem usunąć (pozostałość po tym jak kombinowałem coś jeszcze innego), można usunąć i będzie działać :P

Reply:
Gratulacje, dla mnie mega! Sam bym lepiej nie wymyślił! 

Ano te uwagi Mateusza są słuszne, ale i tak fajnie!

Reply:
Jeśli chcesz zastopować interwal to chyba robi sie to przez

`

function myTimer() {
  var d = new Date();
  var t = d.toLocaleTimeString();
  document.getElementById("demo").innerHTML = t;
}

function myStopFunction() {
  clearInterval(myVar);
}`

Reply:
tak tak, ten mój sposób z ifem jest kiepski, trzeba koniecznie użyć clearInterval